src/Hedgehog/Property.fs

```diff
@@ -254,9 +224,25 @@ module Property =
         reportBool property
         |> Report.render
 
-    let renderBoolWith (config : PropertyConfig) (p : Property<bool>) : string =
-        reportBoolWith config p
-        |> Report.render
+    /// Set the number of times a property is allowed to shrink before the test
+    /// runner gives up and displays the counterexample.
+    let withShrinks (shrinkLimit : int<shrinks>) (Property (config, gen) : Property<'a>) : Property<'a> =
```

Comment:
Might be nice to just take an `int` instead of an `int<shrinks>`

Reply:
Can you expand on this?